src/Base/Base64.h

```diff
@@ -1,39 +1,403 @@
-/* 
-   base64.cpp and base64.h
+/* 

+   base64.cpp and base64.h

+

+   Copyright (C) 2004-2008 René Nyffenegger

+

+   This source code is provided 'as-is', without any express or implied

+   warranty. In no event will the author be held liable for any damages

+   arising from the use of this software.

+

+   Permission is granted to anyone to use this software for any purpose,

+   including commercial applications, and to alter it and redistribute it

+   freely, subject to the following restrictions:

+

+   1. The origin of this source code must not be misrepresented; you must not

+      claim that you wrote the original source code. If you use this source code

+      in a product, an acknowledgment in the product documentation would be

+      appreciated but is not required.

+

+   2. Altered source versions must be plainly marked as such, and must not be

+      misrepresented as being the original source code.

+

+   3. This notice may not be removed or altered from any source distribution.

+

+   René Nyffenegger rene.nyffenegger@adp-gmbh.ch

+

+   Copyright (C) 2019 Zheng Lei (realthunder.dev@gmail.com)
 
-   Copyright (C) 2004-2008 René Nyffenegger
+   * Added support of in-line transformation using boost iostream for memory
+     efficiency
+*/


```

Comment:
Should these ´^M´ artifacts at the end of each comment line be removed?

Reply:
These are different Linux/Windows line endings that are rendered as "^M" by git. Since the world of FreeCAD devs is too evenly split between Windows and Linux, it makes little sense to try to unify, we stopped trying long time ago ;)
This is only cosmetic, and can be disabled in most apps (Github too if I'm not mistaken), so it's not a big problem.

Reply:
A suggestion would have dos2unix running as pre commit hook.